README.md

```diff
@@ -34,6 +34,9 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+
+pip install python-dateutil
```

Comment:
We are using Poetry for managing the project dependencies. So if you want to download a new library, use `poetry add ...` as you did for "requests".

But it looks like this is unused anyway, so just remove this line from the README.

What you could mention in your README is that you need to fill in .env with valid values for the 3 trello variables, and your board needs "To Do" and "Done" column.